apps/api/app/publish_api.py

```
@@ -0,0 +1,487 @@
+        },
+        retry_count=0,
+        created_at=_now(),
+        updated_at=_now(),
+    )
+    session.add(job)
+    session.commit()
+    session.refresh(job)
+
+    try:
+        job.task_id = _dispatch_publish_task(job)
+    except Exception as exc:  # pragma: no cover
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)